packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2865,6 +2924,30 @@ export class ContainerRuntime
 			});
 		}
 
+		let idRange: IdCreationRange | undefined;
+		// Resubmited ops will already have an idRange specified and taking the next creation range
```

Comment:
This comment does not tell if this code takes care of this issue or its telling that the problem can still happen?

Reply:
Also, I don't see any special logic in reSubmit of `idAllocation` op. Is that fine?

Reply:
What will happen if ops are resubmitted? Looks like it will call `takeNextCreationRange` again and its old range will be overwritten.

Reply:
You were right that there was a bug here. I have a resubmission test now and have fixed the resubmission issue by not generating `IdAllocation` ops when an `IdAllocation` op is submitted (resubmit case).